docs/user_guide/2_software_description.md

```diff
@@ -656,9 +661,9 @@ There are two clases available:
 
 The main purpose of the robot commander is to provide a base class to the
 hand commander. The RobotCommander should not be used directly unless necessary.
-Use the `SrHandCommander <HandCommander.html>`__ instead.
+Use the ``SrHandCommander`` instead.
```

Comment:
are double " ` " on each side necessary?

Reply:
they are interpreted in rst to make the text rendered as monospaced text